README.md

```diff
@@ -4,7 +4,11 @@ Tools to construct and process web graphs from Common Crawl data
 
 ## Compiling and Packaging Java Tools
 
-The Java tools are compiled and packaged by [Maven](http://maven.apache.org/). If it's installed just run `mvn package`.
+The Java tools are compiled and packaged by [Maven](https://maven.apache.org/). If Maven is installed just run `mvn package`. Now the Java tools can be run via
+```
+java -cp target/cc-webgraph-0.1-SNAPSHOT-jar-with-dependencies.jar <classname> <args>...
```

Comment:
I had to run:
`WG="java -Xmx12g -cp target/cc-webgraph-0.1-SNAPSHOT-jar-with-dependencies.jar"`
 because when I attempted:
`$WG it.unimi.dsi.webgraph.algo.StronglyConnectedComponents --renumber --sizes cc-main-2020-feb-mar-may-domain` it gave me memory error, when I used `WG="java -cp target/cc-webgraph-0.1-SNAPSHOT-jar-with-dependencies.jar"`.

Reply:
Yes, of course, that's easily possible as the graphs alone are over 4 GB in size. Please add this to the commands, ev. with a comment about the Java heap size.